EMULATOR/LIBETC.C

```diff
@@ -25,6 +31,7 @@ int VSync(int mode)
 		{
 			vsync_callback();
 		}
+		Emulator_EndScene();
```

Comment:
These changes look like the old vsync code. I recommend waiting until it's properly implemented rather than re-adding a hack.

Reply:
The emulator uses new time step code to resolve this hack.